examples/asr/asr_adapters/eval_asr_adapter.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+# Evaluate an adapted model
+
+python eval_asr_adapter.py \
+    --config-path="conf/" \
+    --config-name="asr_adaptation.yaml" \
+    model.pretrained_model=null \
+    model.nemo_model=null \
+    model.adapter.adapter_name=<name of the adapter to evaluate> \
+    model.test_ds.manifest_filepath="<Path to validation/test manifest>" \
+    model.test_ds.batch_size=16 \
+    model.train_ds.manifest_filepath=null \
+    model.validation_ds.manifest_filepath=null \
+    model.adapter.in_features=null \
+    trainer.devices=1 \
+    trainer.precision=32
+
+# Pretrained Models
+
+For documentation on existing pretrained models, please visit -
+https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/asr/results.html
+
+"""
+
+import pytorch_lightning as pl
+from omegaconf import OmegaConf, open_dict
+
+from nemo.collections.asr.models import ASRModel
+from nemo.core.config import hydra_runner
+from nemo.utils import logging
+from nemo.utils.exp_manager import exp_manager
+
+
+def update_encoder_config_to_support_adapter(model_cfg, new_cfg):
+    with open_dict(model_cfg):
+        if 'Adapter' not in model_cfg.encoder._target_:
+            model_cfg.encoder._target_ = model_cfg.encoder._target_ + 'Adapter'
+
+
+def update_model_cfg(original_cfg, new_cfg):
+    with open_dict(new_cfg):
+        # drop keys which dont exist in old config
+        new_keys = list(new_cfg.keys())
+        for key in new_keys:
+            if key not in original_cfg:
+                new_cfg.pop(key)
+                print("Removing unavailable key from config :", key)
```

Comment:
nit: do we have conformity of only using `f-string` formatting syntax in NeMo? I saw mixed use cases.